coalib/output/Logging.py

```diff
@@ -60,15 +62,21 @@ def configure_logging():
         'formatters': {
             'colored': {
                 '()': 'colorlog.ColoredFormatter',
-                'format': '%(log_color)s[%(levelname)s]%(reset)s[%(asctime)s]'
-                          ' %(message)s',
+                'format': '%(log_color)s[%(levelname)s]%(reset)s'
+                          '[%(asctime)s] %(message)s',
                 'datefmt': '%X',
                 'log_colors': {
                     'ERROR': 'red',
                     'WARNING': 'yellow',
                     'INFO': 'blue',
                     'DEBUG': 'green'
                 }
+            },
+            'plain': {
+                '()': 'colorlog.ColoredFormatter',
```

Comment:
I believe you can remove this line, and it will use the default formatter of the logging package.
The default should be able to handle the format string as well, no need to use a custom one in this case.